src/backend/vulkan/src/device.rs

```diff
@@ -212,54 +203,45 @@ impl GraphicsPipelineInfoBuf {
             }
         };
 
-        this.rasterization_state = vk::PipelineRasterizationStateCreateInfo {
-            s_type: vk::StructureType::PIPELINE_RASTERIZATION_STATE_CREATE_INFO,
-            p_next: ptr::null(),
-            flags: vk::PipelineRasterizationStateCreateFlags::empty(),
-            depth_clamp_enable: if desc.rasterizer.depth_clamping {
+        this.rasterization_state = vk::PipelineRasterizationStateCreateInfo::builder()
+            .flags(vk::PipelineRasterizationStateCreateFlags::empty())
+            .depth_clamp_enable(if desc.rasterizer.depth_clamping {
                 if device.shared.features.contains(Features::DEPTH_CLAMP) {
```

Comment:
we don't really need to check for this, can just call `depth_clamp_enable(desc.rasterizer.depth_clamping)`